criteria/active-community.md

```diff
@@ -0,0 +1,44 @@
+---
+order: 15
+---
+# Foster an active community
+
+## Requirements
+
+* Software MUST be in use by multiple parties
+* Communication channels MUST exist for users AND developers
+* The roadmap SHOULD be influenced by the needs of multiple parties
+* MAY hold regular events
+
+## Why this is important
+
+* Members of active communities support each other.
+* Codebases without communities are either too narrow for stewardship by the Foundation for Public code or must continue under incubation until a self-sustaining community can be built.
+
+## What this does not do
+
+* This does not shift responsibility for documentation, support, bug-fixing, or anything else to an unpaid community.
+
+## How to test
```

Comment:
The 'How to test' section should explain how to test for compliance on the 'Requirements', the current things below, whilst worthwhile metrics are not tests. Perhaps they should find a home under business management? Do we want the business management do this?